### ISSUE TITLE
Turn server does not seem to work.

### ISSUE THREAD

**Issue by Beiri22** [bigbluebutton/bbb-install#599]
I just installed the current version of bbb with the bbb-install-2.6-script. When connecting to a conference with firefox, ICE errors occur whenever trying to use audio or video.
Is there any way to easily check if the newly embedded coturn is working correctly?

**Comment by ffdixon** [comment]
Does Chrome work when you connect?

**Comment by Beiri22** [comment]
Yes, Chrome was ok. I therefore suspect the newly integrated Turn server. With the last beta versions I used a separate turn and Firefox did work

**Comment by ffdixon** [comment]
With your separate turn server, if you re-run the `bbb-install-2.6.sh` command with `-c <host>:<secret>`, it will configure BigBlueButton your newly installed BigBlueButton 2.6-rc server to use that separate TURN server.  If so, does it work?

To test your TURN server -- both on the BigBlueButton server and your separate server -- see

 https://docs.bigbluebutton.org./admin/setup-turn-server.html#test-your-turn-server

**Comment by Beiri22** [comment]
I am using terraform to install and use a short term bbb instance. I just reran the installation via terraform to a hetzner vm today and both, firefox and chrome worked. The only thing that I changed was the subdomain. That should not affect any of the functions. I've no clue, why it did not work yesterday. I will watch the behaviour...
